test/pop_protocol_test.erl

```diff
@@ -83,13 +99,21 @@ basic_test() ->
     lists:foldl(FoldFn, PD0, [B1]),
     lists:foldl(FoldFn, PD0, [B1, B3]),
     lists:foldl(FoldFn, PD0, [B1, B2, B3, B4, B5]),
+
+    PD01 = lists:foldl(FoldFn, PD0, [B1]),
```

Comment:
maybe time to a blockmaker function that does this fold :D
maybe something I should do D:

Reply:
pf, good enough :P